model_api/cpp/models/src/image_model.cpp

```diff
@@ -115,6 +115,18 @@ ImageModel::ImageModel(std::shared_ptr<InferenceAdapter>& adapter)
     }
 }
 
+void ImageModel::updateModelInfo() {
+    ModelBase::updateModelInfo();
+
+    model->set_rt_info(useAutoResize, "model_info", "auto_resize");
```

Comment:
this is what we should rename to `embed_preprocessing` as in Python. `embeded_processing` sounds better to me since we deal in postprocessing in some cases.